src/lib/models/events-group.ts

```diff
@@ -41,6 +53,12 @@ const isActivityScheduledEvent = (
   return event.eventType === 'ActivityTaskScheduled';
 };
 
+const isChildWorkflowInitializedEvent = (
```

Comment:
This can happen later, but you can see a world where we break these out into their own file because we might want to use them elsewhere.